drivers/input/input_stmpe811.c

```diff
@@ -111,14 +112,13 @@ LOG_MODULE_REGISTER(stmpe811, CONFIG_INPUT_LOG_LEVEL);
 #define STMPE811_TSC_FRACT_XYZ_CONF 1
 
 struct stmpe811_config {
+	struct input_touchscreen_common_config common;
```

Comment:
Ditto on comment about make sure this structure is first in the struct declaration

Reply:
As with ft5336, we went with the `INPUT_TOUCH_STRUCT_CHECK` macro here instead of a comment